forch/forchestrator.py

```diff
@@ -208,9 +208,16 @@ def _initialize_orchestration(self):
             process_device_behavior = None
             get_vlan_from_segment = None
 
-        self._port_state_manager = PortStateManager(
-            process_device_placement, process_device_behavior, get_vlan_from_segment,
-            self._config.orchestration.sequester_config.segment)
+        if self._metrics:
+            update_device_state_varz = (lambda mac, state: self._metrics.update_var(
+                'device_state', state, labels=[mac]))
+            self._port_state_manager = PortStateManager(
```

Comment:
Refactor this out so constructor call isn't duplicated

Reply:
Done